src/frontend/app.ts

```diff
@@ -137,42 +138,74 @@ export default (octokitApp: App): Express => {
     csrfProtection,
     oauth.checkGithubAuth,
     getGitHubSetup,
+    elapsedTimeMetrics('/github/setup'),
+  );
+
+  app.post(
+    '/github/setup',
+    csrfProtection,
+    postGitHubSetup,
+    elapsedTimeMetrics('/github/setup'),
```

Comment:
I'm fairly certain that this won't work. Your timer start when the app initializes, and then only checks the different when the result is sent.  It should be before the function you want to check the time of and only start the timer when it gets called through express before you call `next()`

Reply:
Also, did the library I linked to didn't work?

Reply:
I couldn't find the metrics in SFX. Additionally, the package itself says `This middleware can be used either globally or on a per-route basis (preferred)`  and then gives the example of 

```function statsd (path) {
  return function (req, res, next) {
    var method = req.method || 'unknown_method';
    req.statsdKey = ['http', method.toLowerCase(), path].join('.');
    next();
  };
}
 
app.get('/', statsd('home'), function (req, res) {
  res.send('Hello World!');
});
 ```
 
 which doesn't use the actual package. I figured it didn't make sense to install a package when we could basically use the example middleware function.

Reply:
oh, thought it was integrated with the library... how disappointing..